ansible/playbook.yml

```diff
@@ -1,9 +1,20 @@
 - hosts: localhost
   connection: local
   become: false
+  vars_prompt:
+    - name: package_install
+      prompt: Would you like to install the require packages[y/n]
+      default: y
+      private: false
+  pre_tasks:
+    - name: Verify OS
+      ansible.builtin.fail:
+        msg: Only Ubuntu 22.04 is supported for this branch. Please refer to https://tier4.github.io/CARET_doc/latest/installation/installation/
+      when: ansible_distribution_version != '22.04'
+
   roles:
-    - { role: lttng, tags: [lttng] }
-    - { role: ros2, rosdistro: humble, tags: [ros2] }
-    - { role: ros-tracing, rosdistro: humble, tags: [ros-tracing] }
-    - { role: visualizer, tags: [visualizer] }
-    - { role: caret, rosdistro: humble, ubuntu_version: 22.04, tags: [caret] }
+    - { role: lttng, tags: [lttng], when: package_install == 'y' }
+    - { role: ros2, rosdistro: humble, tags: [ros2], when: package_install == 'y' }
```

Comment:
@takam5f2 
- When a user already has their own ROS 2 environment, do ROS 2 packages need to be installed?
- If not, I'm wondering whether we should have two `package_install` options:
    - option to install `ros2` only
    - option to install CARET stuffs ( `lttng`, `ros-tracing` , `visualizer` )

- Positive side for separating option is that a user can keep the original ROS 2 environment, while installing libraries needed by CARET
- Negative side for separating option is that the number of options increases and it may be annoying
- What do you think?

Reply:
@takam5f2 
Thank you for fixing it, but I'd like to postpone merging this PR until the planned experiment is finished (maybe tomorrow).